src/protagonist/Engine/Ingest/Image/ImageServer/Clients/CantaloupeThumbsClient.cs

```diff
@@ -84,6 +86,18 @@ public class CantaloupeThumbsClient : IThumbsClient
         throw new HttpException(response.StatusCode, "failed to retrieve data from the thumbs processor");
     }
 
+    private void AttemptToAddStickinessCookie(HttpResponseMessage response)
+    {
+        if (!cantaloupeClient.DefaultRequestHeaders.Contains("Cookie"))
+        {
+            var hasCookie = response.Headers.TryGetValues("Set-Cookie", out var cookies);
```

Comment:
Do we want to grab _all_ cookies returned or would it be better to have a whitelist?